experiment_manager.py

```diff
@@ -323,7 +330,11 @@ def wait_result(self, job):
         return result
 
     def one_dimensional_job(self, config: Config):
+        # input verification
```

Comment:
Comments like this should be written in Imperative tense.

Reply:
fixed